lib/libutee/arch/arm/sub.mk

```diff
@@ -2,4 +2,5 @@ cppflags-y += -I$(sub-dir)/../..
 
 srcs-y += user_ta_entry.c
 srcs-y += utee_misc.c
-srcs-y += utee_syscalls_asm.S
+srcs-$(CFG_ARM32_user_ta) += utee_syscalls_a32.S
+srcs-$(CFG_ARM64_user_ta) += utee_syscalls_a64.S
```

Comment:
So far only 32-bit compilation is supported, here we're just providing some prerequisites (which I have verified with other patches that they are OK).

I'll update the commit message.